src/DocumentationBrowserViewExtension/DocumentationBrowserViewModel.cs

```diff
@@ -186,6 +223,112 @@ private void HandleLocalResource(OpenDocumentationLinkEventArgs e)
             this.shouldLoadDefaultContent = false;
         }
 
+        private void WatchMdFile(string mdLink)
+        {
+            var watcher = packageManagerDoc.GetFileSystemWatcher(mdLink);
+            watcher.Changed += OnCurrentMdFileChanged;
+        }
+
+        private void UnsubscribeMdWatcher(Uri link)
+        {
+            if (link is null)
+                return;
+
+            var watcher = packageManagerDoc.GetFileSystemWatcher(link.OriginalString);
+            if (watcher is null)
+                return;
+
+            watcher.Changed -= OnCurrentMdFileChanged;
+        }
+
+        private void OnCurrentMdFileChanged(object sender, FileSystemEventArgs e)
+        {
+            if (e.FullPath != link.OriginalString)
+                return;
+            if (!(openDocumentationLinkEventArgs is OpenNodeAnnotationEventArgs))
+                return;
+
+            var nodeAnnotationArgs = openDocumentationLinkEventArgs as OpenNodeAnnotationEventArgs;
+            this.content = CreateNodeAnnotationContent(nodeAnnotationArgs);
+            OnContentChanged();
+        }
+
+        private string CreateNodeAnnotationContent(OpenNodeAnnotationEventArgs e)
```

Comment:
think these methods should move to a new `static class NodeDocsMarkdownLoader`
- `private` CreateNodeAnnotationContent
- `private` CreateHeader
- `private` CreateNodeInfo

this new class could expose 2 simple methods :
- `static string FromFile(string mdFilepath)`
- `static string FromString(string mdString)`